server/utils/assessments/utils.ts

```diff
@@ -478,6 +484,23 @@ const acctAlertsFromAssessment = (assessment: Assessment) =>
 const oasysInformationFromAssessment = (assessment: Assessment): OASysSections =>
   assessment.application?.data?.['oasys-import'] || {}
 
+const oasysQuestions = (section: Array<OASysQuestion>) =>
```

Comment:
I wonder if we could stick the oasys-related stuff in it's own oasysUtils file, as this one is getting pretty big now